src/main/scala/goatrodeo/omnibor/strategies/Maven.scala

```diff
@@ -157,7 +157,10 @@ case class MavenState(
                 val manifestString =
                   Helpers.slurpInputToString(manifest.asStream())
                 val props = java.util.Properties()
-                props.load(manifest.asStream())
+                // if this causes an exception, welp...
+                Try {
```

Comment:
Would it be worth logging, or would there be nothing for the user to do anyway?

Reply:
Maybe... do we have a plan in the Ingestion pipeline to review `Error` and other anomalies?